sdn_tests/pins_ondatra/infra_deps.bzl

```diff
@@ -0,0 +1,337 @@
+        patches = ["//:bazel/patches/p4lang.patch"],
+        patch_args = ["-p1"],
+    )
+
+    go_repository(
+        name = "com_github_openconfig_testt",
+        importpath = "github.com/openconfig/testt",
+        commit = "efbb1a32ec07fa7f0b6cf7cda977fa1c584154d6",
+    )
+
+    go_repository(
+        name = "in_gopkg_yaml_v2",
+        importpath = "gopkg.in/yaml.v2",
+        repo_mapping = repo_map,
+        sum = "h1:D8xgwECY7CYvx+Y2n4sBz93Jn9JRvxdiyyo8CTfuKaY=",
+        version = "v2.4.0",
+    )
+
+    go_repository(
+        name = "io_opencensus_go",
+        importpath = "go.opencensus.io",
+        repo_mapping = repo_map,
+        sum = "h1:y73uSU6J157QMP2kn2r30vwW1A2W2WFwSCGnAVxeaD0=",
+        version = "v0.24.0",
+    )
+
+    go_repository(
+        name = "org_golang_google_grpc",
+        importpath = "google.golang.org/grpc",
+        repo_mapping = repo_map,
+        sum = "h1:EhTqbhiYeixwWQtAEZAxmV9MGqcjEU2mFx52xCzNyag=",
+        version = "v1.54.0",
+    )
+
+    go_repository(
+        name = "org_golang_google_grpc_cmd_protoc_gen_go_grpc",
+        importpath = "google.golang.org/grpc/cmd/protoc-gen-go-grpc",
+        repo_mapping = repo_map,
+        sum = "h1:M1YKkFIboKNieVO5DLUEVzQfGwJD30Nv2jfUgzb5UcE=",
+        version = "v1.1.0",
+    )
+
+    go_repository(
+        name = "org_golang_google_protobuf",
+        importpath = "google.golang.org/protobuf",
+        repo_mapping = repo_map,
+        sum = "h1:kPPoIgf3TsEvrm0PFe15JQ+570QVxYzEvvHqChK+cng=",
+        version = "v1.30.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_exp",
+        importpath = "golang.org/x/exp",
+        commit = "aacd6d4b4611949ff7dcca7a0118e9312168a5f8",
+    )
+
+    go_repository(
+        name = "org_golang_x_net",
+        importpath = "golang.org/x/net",
+        repo_mapping = repo_map,
+        sum = "h1:aWJ/m6xSmxWBx+V0XRHTlrYrPG56jKsLdTFmsSsCzOM=",
+        version = "v0.9.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_sync",
+        importpath = "golang.org/x/sync",
+        repo_mapping = repo_map,
+        tag = "v0.3.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_sys",
+        importpath = "golang.org/x/sys",
+        repo_mapping = repo_map,
+        sum = "h1:3jlCCIQZPdOYu1h8BkNvLz8Kgwtae2cagcG/VamtZRU=",
+        version = "v0.7.0",
+    )
+
+    go_repository(
+        name = "org_golang_x_text",
+        importpath = "golang.org/x/text",
+        repo_mapping = repo_map,
+        sum = "h1:2sjJmO8cDvYveuX97RDLsxlyUxLl+GHoLxBiRdHllBE=",
+        version = "v0.9.0",
+    )
+
+
+    git_repository(
+        name = "com_google_googleapis",
+        remote = "https://github.com/googleapis/googleapis",
+        commit = "c4915db59896a1da45b55507ece2ebc1d53ef6f5",
+        shallow_since = "1642638275 -0800",
+    )
+
+    go_repository(
+        name = "com_github_jstemmer_go_junit_report_v2",
+        importpath = "github.com/jstemmer/go-junit-report/v2",
```

Comment:
@VSuryaprasad-HCL Getting a 404 error on this url ? Maybe we haven't added the build targets that exercise this to cause a failure ?
cc: @parth-opensrc

Reply:
Try removing it, if the build works, it should be fine to remove the dep.
If it doesn't, you can try changing the rule:
```
go_repository(
    name = "com_github_jstemmer_go_junit_report_v2",
    commit = "add_commit_hash", 
    importpath = "github.com/jstemmer/go-junit-report/v2",
    remote = "https://github.com/jstemmer/go-junit-report",
    vcs = "git"
)
```

Reply:
Tried removing the dep, The build was successful. Attaching the build info in the chat box.

         INFO: Analyzed 0 targets (1 packages loaded, 0 targets configured).
         INFO: Found 0 targets...
         INFO: Elapsed time: 2.596s, Critical Path: 0.02s
         INFO: 1 process: 1 internal.
         INFO: Build completed successfully, 1 total action